bin/smash

```diff
@@ -225,10 +219,8 @@ def main():
          
             log_quiet('running %s in %s as %s testset #%s %s',t,config.rundir,config.oc_account_name,i,repr(ts)) # log_quiet
 
-         reporter.testcase_start(barename(t),j,i,smashbox.no_engine)
-
          if not args.dry_run:
-            run_multiprocessing_engine(config,os.path.abspath(t),reporter)
+            run_multiprocessing_engine(config,os.path.abspath(t))
 
```

Comment:
Please not that i get rid of whole reporter class in "bin/smash" and "multiprocessing_engine", now reporter is "plugable" in utilities.